### DISCUSSION TITLE
Provide multiple author_associations in payload instead of single association

### DISCUSSION THREAD

**Discussion by jd-solanki** [community/community#9629] [Other Feature Feedback, Questions, & Ideas]
Hi,

Currently, the GitHub context payload only provides a single author association of the user. I am creating an action to toggle issue label based on user and I don't want to toggle label on the issue if a comment is not created by either owner & org member. 

The issue is if the org member is also a contributor in repo then I get `author_association` as a contributor. In this scenario I am unable to identify that it is also org member.

This can be resolved using providing multiple author associations like user is contributor & member both.

**Comment by chrispat** [comment]
Thank you for your feedback.

I don't believe this is specific to Actions as we simply pass the webhook payload along to the workflow.  This may make more sense in the [General Feedback](https://github.com/github/feedback/discussions/categories/general-feedback) category as a update to the webhook payloads overall.

**Comment by jd-solanki** [comment]
Thanks.

I moved it to general feedback. Is it find or I should create a new discussion there?
